src/node_jsvmapi.cc

```diff
@@ -204,7 +204,7 @@ namespace v8impl {
     public:
       CallbackWrapperBase(const T& cbinfo)
         : _cbinfo(cbinfo),
-          _cbdata(cbinfo.Data().As<v8::Object>()) {
+          _cbdata(v8::Local<v8::Object>::Cast(cbinfo.Data())) {
```

Comment:
The .As<v8::Object> is doing the cast in its implementation in v8. So I would prefer to not to use template keyword here.